addons/purchase/models/product.py

```diff
@@ -5,6 +5,7 @@
 from odoo import api, fields, models, _
 from odoo.addons.base.models.res_partner import WARNING_MESSAGE, WARNING_HELP
 from odoo.tools.float_utils import float_round
+from dateutil.relativedelta import *
```

Comment:
`from dateutil.relativedelta import relativedelta` as you don't need the rest (MO for example)